movies-app/src/components/GenreSelect/GenreSelect.jsx

```diff
@@ -1,30 +1,54 @@
-import React, { useState, useEffect} from "react";
-import './GenreSelect.css';
+import React from "react";
+import "./GenreSelect.css";
+import { useSearchParams, useLocation } from "react-router-dom";
 
 const GenreSelect = (props) => {
 
-    const onSelectHandler = (event) => {
-      const genre = event.target.innerHTML;
-      props.onSelect(genre) 
+  const location = useLocation();
+  const [searchParams, setSearchParams] = useSearchParams({});
```

Comment:
Nem kell inicializálni
```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
```